RELEASE/scripts/autoscend/quests/level_01.ash

```diff
@@ -0,0 +1,35 @@
+void tootOriole()
+{
+	//Toot Oriole must be visited each ascension to unlock other quests from the council
+	if(get_property("questM05Toot") == "finished")
+	{
+		return;
+	}
+	
+	//do quest
+	visit_url("tutorial.php?action=toot");
+	if(isActuallyEd())
+	{
+		use(item_amount($item[Letter to Ed the Undying]), $item[Letter to Ed the Undying]);
+	}
+	else
+	{
+		use(item_amount($item[Letter From King Ralph XI]), $item[Letter From King Ralph XI]);
+	}
+	//finishing toot quest is not correctly noticed by mafia. r20655 has workaround of correcting this by refreshing quests
+	cli_execute("refresh quests");
+	
+	if(get_property("questM05Toot") == "finished")
+	{
+		use(item_amount($item[Pork Elf Goodies Sack]), $item[Pork Elf Goodies Sack]);
+		council();
+	}
+	else abort("Failed to finish the Toot Oriole quest. This prevents us from getting other quests from council");
+}
+
+void tootGetMeat()
+{
+	auto_autosell(min(5, item_amount($item[hamethyst])), $item[hamethyst]);
+	auto_autosell(min(5, item_amount($item[baconstone])), $item[baconstone]);
+	auto_autosell(min(5, item_amount($item[porquoise])), $item[porquoise]);
```

Comment:
the 5 here seems redundant. the sack can only contain 2 at most of each gem.
this is also likely a bad idea in casual as it would autosell 15 gems there. which is a pretty big waste

Reply:
> the 5 here seems redundant. the sack can only contain 2 at most of each gem.
You can get all 5 of the same gem type when you open the sack. Search reddit you'll see people who think it's amazeballs when it happens.
Shouldn't need to autosell in casual anyway so maybe just leave it as is & condition these lines with `!can_interact()` or something similar?

Reply:
thanks for the suggestion. change made